test/integration/redirects_test.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+require("test_helper")
+
+class RedirectsTest < IntegrationTestCase
+  def test_controller
+    get("/article")
+    assert_equal("/articles", @response.request.fullpath)
```

Comment:
1. Should I use literal paths or instead use path helpers here and elsewhere. I.e.
` assert_equal("/articles", @response.request.fullpath)`
vs
` assert_equal(articles_path, @response.request.fullpath)`
I think the literal path is faster, but I couldn't see a difference in a quick-and-dirty `time` test.
I think the path helper is more idiomatic.
2. `fullpath` or `original_fullpath`? (I don't understand the difference.)